R/pmin-method.R

```diff
@@ -0,0 +1,39 @@
+#'
+#' @title pmin/pmax
+#' @name magpie-pmin-pmax
+#' @aliases pmin pmax pmin,magpie-method pmin,ANY-method, pmax,magpie-method pmax,ANY-method
+#' @description magclass-aware pmin/pmax, which calculate the parallel minima/maxima of the input values
+#' @details
+#' \code{\link[base]{pmin}}/\code{\link[base]{pmax}} do not care about the order of items in a magclass
```

Comment:
I'm unsure, but does this also work?
```suggestion
#' \code{\link{base::pmin}}/\code{\link{base::pmax}} do not care about the order of items in a magclass
```

Reply:
For that to work, we would apparently need to use `\link[base::pmin] `

> There are two other forms with an optional ‘anchor’ argument, specified as \link[pkg]{foo} and \link[pkg:bar]{foo}, to link to topics foo and bar respectively in the package pkg. They are currently only used in HTML help (and ignored for hyperlinks in LaTeX conversions of help pages). One should be careful about topics containing special characters (such as arithmetic operators) as they may result in unresolvable links, and preferably use a safer alias in the same help page.

[https://rstudio.github.io/r-manuals/r-exts/Writing-R-documentation-files.html#cross-references](https://rstudio.github.io/r-manuals/r-exts/Writing-R-documentation-files.html#cross-references)

Reply:
From the quote you send it does not look to me like `\link[base::pmin]` would work, but it is quite confusing, are they saying `\link[base:pmin]{pmin}` would do it? Anyway, it's not really important enough to find out I'd say :see_no_evil: